shared/editor/embeds/GoogleForms.test.ts

```diff
@@ -0,0 +1,55 @@
+import GoogleForms from "./GoogleForms";
+
+describe("GoogleForms", () => {
+  const match = GoogleForms.ENABLED[0];
+  const match2 = GoogleForms.ENABLED[1];
+
+  test("to be enabled on long-form share links", () => {
+    expect(
+      "https://docs.google.com/forms/d/e/1FAIpQLSetbCGiE8DhfVQZMtLE_CU2MwpSsrkXi690hkEDREOvMu8VYQ/viewform?usp=sf_link".match(
+        match
+      )
+    ).toBeTruthy();
+    expect(
+      "https://docs.google.com/forms/d/e/1FAIpQLSetbCGiE8DhfVQZMtLE_CU2MwpSsrkXi690hkEDREOvMu8VYQ/viewform".match(
+        match
+      )
+    ).toBeTruthy();
+    expect(
+      "https://docs.google.com/forms/d/e/1FAIpQLSetbCGiE8DhfVQZMtLE_CU2MwpSsrkXi690hkEDREOvMu8VYQ/viewform?embedded=true".match(
+        match
+      )
+    ).toBeTruthy();
+  });
+
+  test("to be enabled on edit links", () => {
```

Comment:
Not sure if we need this. Open to feedback.

Reply:
👍 Yep